### DISCUSSION TITLE
Reduce cache times?

### DISCUSSION THREAD

**Discussion by TheCableGuy99** [AdguardTeam/AdGuardHome#4669] [Q&A]
Hi,

I regularly make changes to my Cloudflare DNS service for different reasons.

I know Cloudflare updates instantly and when I check why I’m not seeing the changes. Adguard says the record was served from Cache.

Is there a way I can reduce the cache timing right down to 1 minute, so anything cached longer than a minute does a fresh lookup?

Thanks for your help.

**Comment by ainar-g** [comment]
You can use the _Override maximum TTL_ option on the *DNS settings* page.
